cmd/tiller/tiller.go

```diff
@@ -90,44 +102,102 @@ func main() {
 }
 
 func start(c *cobra.Command, args []string) {
+	// Set the log level first
+	logLevel, err := logutil.GetLevel(level)
+	if err != nil {
+		logger.WithFields(log.Fields{
```

Comment:
@nebril Good point. I was going back and forth on doing that because I didn't want to have to define it in each file 😄. I'll go back and do that